sci-geosciences/gmt/gmt-4.5.17.ebuild

```diff
@@ -47,40 +45,37 @@ src_prepare() {
 	mkdir "${S}/share/coast"
 	mv -f "${WORKDIR}/${GSHHG}/"*.nc "${S}/share/coast/" || die
 
-	tc-export AR RANLIB
-
-	autotools-utils_src_prepare
-
-	eautoreconf
+	default
 }
 
 src_configure() {
-	local myeconfargs=(
-		--libdir=/usr/$(get_libdir)/${P}
-		--includedir=/usr/include/${P}
-		--datadir=/usr/share/${P}
-		--docdir=/usr/share/doc/${PF}
-		--disable-update
-		--disable-matlab
-		--disable-xgrid
-		--disable-debug
-		$(use_enable gdal)
-		$(use_enable netcdf)
-		$(use_enable octave)
-		$(use_enable debug devdebug)
-		$(use_enable !metric US)
-		$(use_enable postscript eps)
-		$(use_enable mex)
+	econf \
+		--libdir=/usr/$(get_libdir)/${P} \
+		--includedir=/usr/include/${P} \
+		--datadir=/usr/share/${P} \
+		--docdir=/usr/share/doc/${PF} \
+		--disable-update \
+		--disable-matlab \
+		--disable-xgrid \
+		--disable-debug \
+		$(use_enable gdal) \
+		$(use_enable netcdf) \
+		$(use_enable octave) \
+		$(use_enable debug devdebug) \
+		$(use_enable !metric US) \
+		$(use_enable postscript eps) \
+		$(use_enable mex) \
 		$(use_enable gmttria triangle)
-		)
-	autotools-utils_src_configure
 }
 
 src_install() {
-	autotools-utils_src_install install-all
+	emake DESTDIR="${D}" install-all || die
```

Comment:
no die

Reply:
Done. As I understand it, "die" is not necessary, because the errors are handled inside the emake?